allennlp/models/semantic_role_labeler.py

```diff
@@ -191,8 +198,43 @@ def get_viterbi_pairwise_potentials(self):
 
     @classmethod
     def from_params(cls, vocab: Vocabulary, params: Params) -> 'SemanticRoleLabeler':
-        text_field_embedder = TextFieldEmbedder.from_params(vocab, params.pop("text_field_embedder"))
-        stacked_encoder = Seq2SeqEncoder.from_params(params.pop("stacked_encoder"))
+        """
+        With an empty ``params`` argument, this will instantiate a SRL model with the same
+        configuration as published in the Deep Semantic Role Labeling - What works and what's
+        next  paper, as long as you've set ``allennlp.common.constants.GLOVE_PATH`` to the
+        location of your gzipped 100-dimensional glove vectors.
+
+        If you want to change parameters, the keys in the ``params`` object must match the
+        constructor arguments above.
+        """
+        default_embedder_params = {
```

Comment:
I'm thinking there's got to be a better way to do this.  We're going to want to have configuration files that specify training datasets and model parameters for these models, and we'll be duplicating ourselves here and there.  We'll definitely want the full model parameters specified there, to make the connection to the data loading obvious, and just for ease of someone browsing through the configuration files.

What if we allow `params` to be `None` here, and if it's `None`, we load the parameters from that file?  This would be changed later, once we actually have such files.